deltachat-ffi/deltachat.h

```diff
@@ -4498,14 +4498,31 @@ int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
 
 
 /**
- * This event is sent out to the inviter when a joiner successfully joined a group.
+ * (DEPRECATED)
```

Comment:
maybe we can just remove DC_EVENT_SECUREJOIN_MEMBER_ADDED completely, did a quick look, it seems not to be used in android/ios/desktop. it was also renamed recently without much troubles (https://github.com/deltachat/deltachat-core-rust/commit/76610f1e724db1eaccd18eecf43eb61838d18e88)

Reply:
k, i will replace DC_EVENT_SECUREJOIN_MEMBER_ADDED with DC_EVENT_MEMBER_ADDED.  One can check with the chat if this was a verified group.